tests/test_machine.py

```diff
@@ -5,64 +5,51 @@
 
 import pytest
 
-import mkosi.machine as machine
 from mkosi.backend import MkosiException
+from mkosi.machine import Machine, MkosiMachineTest
 
 pytestmark = [
     pytest.mark.integration,
-    pytest.mark.parametrize("verb", ["boot", "qemu"]),
     pytest.mark.skipif(os.getuid() != 0, reason="Must be invoked as root.")
 ]
 
 
-def test_simple_run(verb: str) -> None:
-    with machine.Machine([verb]) as m:
-        p = m.run(["echo", "This is a test."])
-        assert "This is a test." == p.stdout.strip("\n")
+class MkosiMachineTester(MkosiMachineTest, args=["boot"]):
```

Comment:
`MkosiMachineTester` is a weird name for a class. Maybe go for `MkosiMachineTestCase` which matches `unittest.TestCase` more closely?